tests/snapd

```diff
@@ -30,15 +23,5 @@ su ${TEST_USER} -c "${LXC} info"
 echo "==> Deleting test user"
 userdel -r ${TEST_USER} 2>/dev/null || true
 
-echo "==> Deleting network device"
```

Comment:
can you keep this, but change it to clean up the resources that lxd-user should have created, that way if they dont exist we can detect the regression.

Reply:
That's not my understanding of what the test does. Since the `${TEST_USER}` is made a member of the `lxd` group, it's not relying on `lxd-user`.

I'm happy to extend that to also test `lxd-user` but want to check with you first.

Reply:
But since you've mentioned `lxd-user`, I want it to be covered as well, at the very least for code coverage reasons ;)
I'll fill up a Jira card to that effect.